cms/core/templates/wagtailadmin/pages/page_listing_header.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'wagtailadmin/pages/page_listing_header.html' %}
+{% load wagtailadmin_tags %}
+
+{% block breadcrumbs %}
+    {{ block.super }}
+    {% if parent_page.url %}
```

Comment:
question: why the URL check? note that this incurs additional queries as .url is a property linked to `get_url` ([ref](https://github.com/wagtail/wagtail/blob/main/wagtail/models/pages.py#L1416))

Reply:
This was so that the label doesn't appear when you are at the root level, but maybe there is a more elegant solution?

Reply:
`if not page.is_root`

Reply:
For some reason I thought I tried that already... Thanks! [c5ab360](https://github.com/ONSdigital/dis-wagtail/pull/170/commits/c5ab3607245ab1aa408799e8c151c6080a380695)